src/syrupy/report.py

```diff
@@ -41,16 +43,24 @@ class SnapshotReport(object):
     update_snapshots: bool = attr.ib()
     warn_unused_snapshots: bool = attr.ib()
     assertions: List["SnapshotAssertion"] = attr.ib()
+    targeted_items: Optional[Set[Any]] = attr.ib()
     discovered: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     created: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     failed: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     matched: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     updated: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
     used: "SnapshotFossils" = attr.ib(factory=SnapshotFossils)
 
+    def __filter(self, item: SnapshotFossil) -> bool:
```

Comment:
(style) I'll add this to the contributing guide too. For consistency we previously decided to use forward references for all class types.

```suggestion
    def __filter(self, item: "SnapshotFossil") -> bool:
```